transcom/mymove
# Get move documents endpoint/handlers and display in office document viewer

## Description

Adds an endpoint to retrieve all documents associated to a move that provides their titles, and status and way to get document contents. The document viewer displays the documents by title and status.


To test:

* run `make e2e_test`

OR 

* With a move in the db, create a move document: go to `http://officelocal:3000/moves/{move_id}/documents`, then click on "upload new document" link in the column on the right. Follow instructions to upload a new move document.
* enter the URL above in the browser and refresh to see new move documents.

Open questions:
* what should the date of the upload in the wireframe be based off of? Currently not being shown...

## Code Review Verification Steps

* [x] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/blob/master/docs/backend.md#logging)
 [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* [x] (TRIAL) Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/158768250) for this change

## Screenshots
<img width="400" alt="screen shot 2018-07-11 at 4 01 41 pm" src="https://user-images.githubusercontent.com/8170735/42603781-14588d18-8524-11e8-9c96-2870b5e45e60.png">
